MerchantSDK/GiniMerchantSDK/Sources/GiniMerchantSDK/Core/PaymentReviewViewController+UICollection.swift

```diff
@@ -1,42 +0,0 @@
-//
-//  PaymentReviewViewController+UICollection.swift
-//  GiniMerchantSDK
-//
-//  Copyright © 2024 Gini GmbH. All rights reserved.
-//
-
-import UIKit
-import GiniUtilites
-
-// MARK: - UICollectionViewDelegate, UICollectionViewDataSource
-
-extension PaymentReviewViewController: UICollectionViewDelegate, UICollectionViewDataSource, UICollectionViewDelegateFlowLayout {
```

Comment:
Clarification: We will still need these in next phase when all these will go in Payment Component SDK that will be used commonly by GiniHealthSDK and GiniMerchantSDK